### DISCUSSION TITLE
genetic inheritence with Synthea

### DISCUSSION THREAD

**Discussion by s1111809** [synthetichealth/synthea#1252] [Q&A]
Hii all,

I have a question regarding genetic inheritance. I am looking into Synthea for genetic inheritance regarding diseases. I believe Synthea drops a baby in the population pool with random generated statistics such as weight. However, I saw a paper (https://www.mdpi.com/2079-9292/11/8/1199) where they were able to generate a file with Ancestral allele, Variant allele and the allele of the generated person. I also saw a comment from a person (https://github.com/synthetichealth/synthea/pull/186#issuecomment-294845730) who apparently successfully ran a population.

My question is as follows: Where or how do I enable something that makes the babies have inherited genes from preferably both parents? I might completely look over it, but I can't find it.

Thanks in advance!
Alex

**Comment by jawalonoski** [comment]
That code doesn't exist in the `master` branch of Synthea, it exists in the `genetics-integration` branch (and possibly others, I didn't check).

See https://github.com/synthetichealth/synthea/compare/master...genetics-integration
